File: cpp/apcs/1/c294/c294.cpp
#include<iostream>
#include<algorithm>

using namespace std;
int main(){
    short l[3];
    cin>>l[0]>>l[1]>>l[2];
    sort(l,l+3);
    cout<<l[0]<<' '<<l[1]<<' '<<l[2]<<endl;
    if(l[0]+l[1]<=l[2]){
        cout<<"No";
    }
    else if(l[0]*l[0]+l[1]*l[1]<l[2]*l[2]){
        cout<<"Obtuse";
    }
    else if(l[0]*l[0]+l[1]*l[1]==l[2]*l[2]){
        cout<<"Right";
    }
    else{
        cout<<"Acute";
    }
}

File: python/apcs/1/c294.py
a,b,c=sorted([int(x) for x in input().split()])
print(a,b,c)
if a+b>c:
    if a**2+b**2<c**2:
        print('Obtuse')
    elif a**2+b**2==c**2:
        print('Right')
    elif a**2+b**2>c**2:
        print('Acute')
else:
    print('No')